actions/FirefallClient.test.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import { FirefallClient } from './FirefallClient.js';
+import { wretchRetry } from './Network.js';
+
+const { WretchError } = require('wretch');
+
+jest.mock('./Network.js');
+
+const firefall = new FirefallClient('endpoint', 'apiKey', 'org', 'accessToken');
+let error;
+
+beforeEach(() => {
+  jest.clearAllMocks();
+
+  error = new WretchError('Internal Server Error');
```

Comment:
I would avoid using this dangling mutable variable and replace it with a factory function like this:

```js
function createWretchError(status) {
   const error = new WretchError('Internal Server Error');
   error.status = status;
   return error;
}
```